src/main.ts

```diff
@@ -234,6 +235,12 @@ export default class ImporterPlugin extends Plugin {
 				importer: OneNoteImporter,
 				helpPermalink: 'import/onenote',
 			},
+			'apple-notes': {
```

Comment:
Move this to the top of the list (it should be alphabetical)

Reply:
Not sure about this, since it only works on Mac, and the one at the top is the first one everyone sees. Maybe we could move it to the bottom, so the ones that work on every platform are alphabetised nicely.

Reply:
Bear is also macOS/iOS only, so I don't see it as a problem. Maybe in the future we can add a generic way for importers to list which platforms they work on, and make that a conditional display based on the user's current platform. Opened #128 for this. In the meantime we can keep the alphabetical order here.